src/modules/feature-modules/innovator/services/innovator.service.ts

```diff
@@ -69,7 +69,7 @@ export class InnovatorService extends CoreService {
 
   declineAction(innovationId: string, actionId: string, body: MappedObjectType): Observable<{ id: string }> {
 
-    const url = new UrlModel(this.API_URL).addPath('innovators/:userId/innovations/:innovationId/actions/:actionId').setPathParams({ userId: this.stores.authentication.getUserId(), innovationId, actionId });
+    const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/actions/:actionId').setPathParams({ userId: this.stores.authentication.getUserId(), innovationId, actionId });
```

Comment:
@manuelsousa-bjss userId does not have to be passed on pathParams

Reply:
nice catch